<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UdpServer</class>
 <widget class="QWidget" name="UdpServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UdpServer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="ip_lab">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>IP:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ip_le">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>20</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="port_lab">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>PORT:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="port_le">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>20</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="start_pb">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>启动</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="recvMsg_te">
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
